compute/instances/create-start-instance/create_instance_test.go

```diff
@@ -199,3 +199,100 @@ func TestComputeCreateInstanceSnippets(t *testing.T) {
 		t.Errorf("deleteDisk got err: %v", err)
 	}
 }
+
+func TestComputeBulkCreateInstanceSnippets(t *testing.T) {
+	//ctx := context.Background()
+	//var r *rand.Rand = rand.New(
+	//	rand.NewSource(time.Now().UnixNano()))
+	//tc := testutil.SystemTest(t)
```

Comment:
please be sure to call `testutil.SystemTest()`, as there are some environment variable checks in there that we want consistently applied to all tests.

it looks like the rest of this was for reference, and can probably be deleted.

Reply:
it is leftovers, removed